lib/chef/dsl/recipe.rb

```diff
@@ -176,6 +176,24 @@ def method_missing(method_symbol, *args, &block)
           raise NameError, "No resource, method, or local variable named `#{method_symbol}' for #{describe_self_for_error}"
         end
       end
+
+      module Everything
```

Comment:
Not sure what the right word is.  This is the full recipe DSL (rather than a partial one, which is what Chef::DSL::Recipe is right now).